data/soapvalidator/MailClient/AddressList/ZCS-5847_CreateAddressList.xml

```diff
@@ -0,0 +1,439 @@
+			1. Create a new address list with search filter to bring
+			bring all members
+			from ldap whose jobTitle is equal to manager and those whose work country
+			is not India
+		</t:steps>
+
+		<t:property name="addressList4_name" value="addressList4_${TIME}" />
+		<t:property name="country" value="India" />
+
+		<t:test>
+			<t:request>
+				<CreateAddressListRequest type="account"
+					xmlns="urn:zimbraAdmin">
+					<name>${addressList4_name}</name>
+					<searchFilter>
+						<conds not="false" or="1">
+							<cond not="0" attr="jobTitle" op="eq" value="${manager}" />
+							<cond not="1" attr="workCountry" op="eq" value="${country}" />
+						</conds>
+					</searchFilter>
+					<domain by="name">${defaultdomain.name}</domain>
+				</CreateAddressListRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:CreateAddressListResponse" />
+			</t:response>
+		</t:test>
+	</t:test_case>
+
+
+	<t:test_case testcaseid="CreateAddressList_05" type="smoke"
+		bugids="ZCS-5847">
+		<t:objective>Create a new address list for invalid domain
+		</t:objective>
+		<t:steps>
+			1. Create a new address list with search filter set and
+			invalid domain
+		</t:steps>
+
+		<t:property name="addressList5_name" value="addressList5_${TIME}" />
+		<t:property name="invalid_domain" value="domain.com" />
+
+		<t:test>
+			<t:request>
+				<CreateAddressListRequest type="account"
+					xmlns="urn:zimbraAdmin">
+					<name>${addressList5_name}</name>
+					<searchFilter>
+						<conds not="false" or="1">
+							<cond not="0" attr="jobTitle" op="eq" value="${manager}" />
+							<cond not="1" attr="workCountry" op="eq" value="${country}" />
+						</conds>
+					</searchFilter>
+					<domain by="name">${invalid_domain}</domain>
+				</CreateAddressListRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//zimbra:Code" match="^service.INVALID_REQUEST" />
+			</t:response>
+		</t:test>
+	</t:test_case>
+
+	<t:test_case testcaseid="CreateAddressList_06" type="smoke"
+		bugids="ZCS-5847">
+		<t:objective>Create a new address list for resource type</t:objective>
+		<t:steps>
+			1. Create a new address list with search filter set for
+			fetching resources
+		</t:steps>
+
+		<t:property name="addressList6_name" value="addressList6_${TIME}" />
+
+		<t:test>
+			<t:request>
+				<CreateAddressListRequest type="resource"
+					xmlns="urn:zimbraAdmin">
+					<name>${addressList6_name}</name>
+					<searchFilter>
+						<conds not="0" or="1">
+							<conds />
+							<cond attr="zimbraCalResType" op="eq" value="equipment" />
+						</conds>
+					</searchFilter>
+					<domain by="name">${defaultdomain.name}</domain>
+				</CreateAddressListRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:CreateAddressListResponse" />
+			</t:response>
+		</t:test>
+	</t:test_case>
+
+	<t:test_case testcaseid="CreateAddressList_07" type="smoke"
+		bugids="ZCS-5847">
+		<t:objective>Create a new address list for all members</t:objective>
```

Comment:
Please make sure objective matches test scenario. Here it is creating AL for based on postal code and resource type.